resources/src/IconActionFilter.tsx

```diff
@@ -0,0 +1,152 @@
+import {Fragment, useContext, useMemo, useState} from "react"
+import {Link, Modal, Pagination, Select, Spacer, Text, Tooltip, useModal} from "@geist-ui/react"
+import {Filter} from "@geist-ui/react-icons"
+import {DataContext, TranslationData, TranslationStatus} from "./Data"
+import {FileAction, tldrPageUrl} from "./GitHubPage"
+import {useEscClose} from './useEscClose'
+
+enum FilterType {
+    Outdated = 'outdated',
+    NotTranslated = 'not-translated'
+}
+
+type FilterAttributes = { type: FilterType, language: string }
+
+type FilteredPage = { os: string, page: string }
+
+const FilterSelection = (props: { initial: FilterAttributes, onChange: (attributes: FilterAttributes) => void }) => {
+    const {data} = useContext(DataContext)
+    const [type, setType] = useState(props.initial.type)
+    const [langauge, setLanguage] = useState(props.initial.language)
+
+    const languageOptions = data?.languages.map(lang =>
+        <Select.Option value={lang} key={lang}>{lang}</Select.Option>
+    ) ?? []
+
+    return <div>
+        Show
+        <Spacer y={.25} inline={true}/>
+        <Select initialValue={props.initial.type} size='small' onChange={(value: FilterType) => {
+            setType(value)
+            props.onChange({type: value, language: langauge})
```

Comment:
```suggestion
            props.onChange({type: value, language})
```

Reply:
I personally kinda like the explicitness and symmetry of the non-abbreviated form. I'd keep this line as it currently is.